test/Microsoft.Identity.Abstractions.Tests/ManagedIdentityDescriptionTests.cs

```diff
@@ -0,0 +1,23 @@
+﻿using Xunit;
+
+namespace Microsoft.Identity.Abstractions.Tests
+{
+    public class ManagedIdentityDescriptionTests
+    {
+        /// <summary>
+        /// If no field is set for <see cref="ManagedIdentityDescription"/> the <see cref="ManagedIdentityDescription.IdType"/>
+        /// field needs to default to <see cref="ManagedIdentitySource.SystemAssigned"/> as other Microsoft.Identity libraries
+        /// will depend on this.
+        /// </summary>
+        [Fact]
+        public void ManagedIdentity_NoDescriptionFieldsSet()
+        {
+            // Arrange
+            ManagedIdentityDescription description = new();
+
+            // Assert
+            Assert.Equal(ManagedIdentityType.SystemAssigned, description.IdType);
```

Comment:
Does it work? The attribute is not enough to set the value?

Reply:
This test does work, it just makes sure that even if no values are set the ManagedIdentityDescription object will always have a type because guaranteeing that  will simplify the if we do in Id Web... I'll add some prevention tomorrow to keep anyone from setting it to null themselves.